src/components/layout/OrgHeader.tsx

```diff
@@ -52,7 +44,7 @@ const OrgHeader = ({ org } : OrgHeaderProps) : JSX.Element => {
             </Flex>
             <View>
                 <Heading level={ 1 }>
-                    { org.title }
+                    { org?.title }
```

Comment:
`org` should be required to exist in this component, so there should be no need for this.